CHANGES.md

```diff
@@ -1,3 +1,39 @@
+## edge-20.1.8
```

Comment:
Should be `edge-20.1.1`, unless we're moving to a different versioning schema

Reply:
whoops! thanks for catching that!